.gitignore

```diff
@@ -0,0 +1 @@
+*.db
```

Comment:
Remove from submission

Reply:
THX for the review; I understand its under-appreciated, and time-consuming.

I don't mind making changes however, there is way too many rule inconsistency to keep up with. I can't micromanage every commit; can we generate a single "RULES" list... THX

https://github.com/xbmc/repo-scripts/pull/660#discussion_r151667266


I'm leaving this commit as is, merge or don't:

Reply:
Sorry lunatixz I understand the frustration. Martijn doesn't review addons since ~ 2017 and other people have picked up reviews and the repo curation and opinions differ.

Furthermore, your addon has "Team Kodi" as author, so we can't be sending the wrong message and should play by the book. To be honest, an addon that is authored by team kodi normally means it should be available from github.com/xbmc organization, but I'm not even considering it here as you are a team member.

You shouldn't be micromanaging commits, you've now options to submit the addon automatically and exclude files from the submission using gitattributes (however it is designed for single repo addons - https://github.com/xbmc/action-kodi-addon-submitter)